src/sonic_ax_impl/mibs/ietf/rfc4292.py

```diff
@@ -35,7 +35,12 @@ def update_data(self):
                 ipn = ipaddress.ip_network(ipnstr)
                 ent = self.db_conn.get_all(mibs.APPL_DB, routestr, blocking=True)
                 nexthops = ent[b"nexthop"].decode()
-                for nh in nexthops.split(','):
+                ifnames = ent[b"ifname"].decode()
+                for nh, ifn in zip(nexthops.split(','), ifnames.split(',')):
```

Comment:
according to my observation, it is possible that the number of nexthops and the number of ifnames cannot match. it happens when the ifnames is `eth0` but no nexthops.
```
127.0.0.1:6379> hgetall ROUTE_TABLE:10.3.146.0/23
1) "nexthop"
2) ""
3) "ifname"
4) "eth0"
```